lib/stubscriptutils.js

```diff
@@ -6,13 +6,17 @@ import * as cp from "child_process";
 const tmp = os.tmpdir();
 
 export const extractStubs = async (protoFilePath, output_path) => {
-  const protoc = path.resolve("./node_modules/protoc/protoc/bin/protoc");
-  const plugin = path.resolve("./node_modules/ts-protoc-gen/bin/protoc-gen-ts");
+  const protoc = path.resolve("./node_modules/grpc-tools/bin/protoc");
+  const nodeplugin = path.resolve("./node_modules/.bin/grpc_tools_node_protoc_plugin");
+  const jsplugin = path.resolve("./node_modules/.bin/protoc-gen-ts");
   if (!fs.existsSync(protoc)) {
-    console.log(" Protoc file does not exists ");
+    console.log(" protoc file does not exists ");
   }
-  if (!fs.existsSync(plugin)) {
-    console.log(" Plugin file does not exists ");
+  if (!fs.existsSync(nodeplugin)) {
+    console.log(" Node plugin file does not exists ");
+  }
+  if (!fs.existsSync(jsplugin)) {
+    console.log(" js plugin file does not exists ");
```

Comment:
in case of error. don't you want to stop the process ?
and trigger an alert?
Because, even if the program continues without nodePlugin, there won't be anyfiles. and we will never know the lambda faced an error. 

---
Think about it. and update accordingly.

Reply:
done